randovania/games/dread/gui/ui_files/games_tab_dread_widget.ui

```diff
@@ -183,80 +192,27 @@
      <rect>
       <x>0</x>
       <y>0</y>
-      <width>621</width>
-      <height>1106</height>
+      <width>635</width>
+      <height>470</height>
      </rect>
     </property>
+    <property name="mouseTracking">
+     <bool>true</bool>
+    </property>
     <layout class="QVBoxLayout" name="known_crashes_layout">
      <item>
       <widget class="QLabel" name="known_crashes_label">
+       <property name="mouseTracking">
+        <bool>true</bool>
+       </property>
        <property name="text">
-        <string>Metroid Dread is known to crash in some circumstances. Make sure to read the following list and how to avoid these.
-
-
----
-
-
-## After using warp to start in a Save Station.
```

Comment:
on a side note, fairly certain that we can remove this and the "random ryujinx crashing"
At least i haven't heard of these happening in quite a while.

Reply:
I've had random ryu crashes tho. seems like most of the unresolved things are related to emmi zone jank. or at least, "running around in emmi zone and ryujinx explodes for no reason" is almost always where i get random crashes.

Reply:
gotcha, havent heard anyone report it in the last few months or so so assumed it was an issue that was fixed already.

Reply:
I'll leave these in for now. Can always remove it later.